audit/audit_contract.go

```diff
@@ -182,6 +170,7 @@ func list(it iterator.Iterator) [][]byte {
 
 	ignore := [][]byte{
 		[]byte(netmapContractKey),
+		//TODO(@acid-ant): remove notaryDisabled in future version
 		[]byte(notaryDisabledKey),
```

Comment:
We delete it in `_deploy`, this line can be removed.

Reply:
Thanks, removed.